audio_sync/cli.py

```diff
@@ -90,7 +90,7 @@ def CalculatePercentiles(latencies, percentiles=(0, 50, 75, 90, 95, 99, 100)):
   """
   values = [d for _, d in latencies if not math.isnan(d)]
   if values:
-    vals = numpy.percentile(sorted([abs(v) for v in values]),
+    vals = numpy.percentile(sorted([v for v in values]),
```

Comment:
It's not obvious to me what's the global effect of this change, but this doesn't achieve what the function is intended to calculate (which unfortunately isn't clearly stated in the function's docstring). The percentiles should go from 0% as the min latency (least concerning and barely affecting audio) to 100% as the max (the greatest concerning and more noticeable). Removing the abs is going to change this and I am not sure how this affects the different plots. If you could show some examples that would be great.

Reply:
Actually it doesn't matter anymore. I had this in my code before I added the ASCII plot.